other/install.php

```diff
@@ -1553,7 +1553,10 @@ function AdminAccount()
 		}
 		elseif ($_POST['username'] != '')
 		{
-			$incontext['member_salt'] = substr(md5($smcFunc['random_int']()), 0, 4);
+            if (!is_callable('random_int'))
```

Comment:
The code is not well aligned -> because you use space and the code is using tab here.